src/index.ts

```diff
@@ -123,66 +131,24 @@ export default class CodeTool implements BlockTool {
     };
 
     this.data = {
-      code: data.code || '',
+      code: data.code as string || '',
```

Comment:
casting seems unexpected. `data` is typed as `CodeData` where `code` is a string

Reply:
I was getting lint errors on the same line:
`Unsafe assignment of an `any` value` and 
`Unexpected any value in conditional. An explicit comparison or type cast is required`

Reply:
try using `data.code ?? ''`

Reply:
tried this, but still getting the same error

Reply:
since data.code ?? ' ' didnt work, should I caste it to string or ignore the linting rule for this line ?